### PR TITLE
(tlg0540)_003_004

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Next batch of Lysias files!

### CODE REVIEW

**Comment by lcerrato** [comment]
@AlisonBabeu 
I'm wondering are these just meant to bibliographic changes or are they meant to address other header issues, too?

**Comment by AlisonBabeu** [comment]
hi @lcerrato I'm addressing general header issues like adding in an editor/translator, updating the work title if necessary, as well as updating the structured bibliographic section and adding in page level links. 

I'm not touching any of the sponsor info or attributions issues at the moment.

**Comment by lcerrato** [comment]
@AlisonBabeu Thanks!

File: data/tlg0540/tlg004/tlg0540.tlg004.perseus-grc2.xml [hunk 1]
```diff
@@ -29,14 +30,17 @@
         <biblStruct>
           <monogr>
             <author>Lysias</author>
-            <title>Lysias with an English translation by W.R.M. Lamb, M.A.</title>
-
+            <title>Lysias</title>
+            <editor role="translator">W.R.M. Lamb</editor>
```
Comment (lcerrato) [inline data/tlg0540/tlg004/tlg0540.tlg004.perseus-grc2.xml:34]: not translator here

**Review by lcerrato — CHANGES_REQUESTED** [review]

**Review by lcerrato — APPROVED** [review]

**Comment by AlisonBabeu** [comment]
hey @lcerrato thanks for fixing things, I have no idea how I eliminated the closing work tag in the cts_.xml files. I really miss the STOA XML validation working.

**Pull request merged**